terraform/domains/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    dnsimple = {
+      source  = "dnsimple/dnsimple"
+      version = "1.2.0"
+    }
+  }
+}
+
+provider dnsimple {}
```

Comment:
I think creating variables and using them as part of the config would also be a valid approach. In either case we should include in the readme what env variables need to be set for this to work.